### DISCUSSION TITLE
Batch rename missing?

### DISCUSSION THREAD

**Discussion by ManuelMendozaLevo** [TabularEditor/TabularEditor3#129] [🙋♀️TE3 Q&A] [answered]
Hi all. I´m loving this tool! but I can't find the batch rename option which was previously available on tabular editor 2 (F2 or right click and batch rename).

Am I doing something wrong or it's actually missing from this build.

**Comment by otykier** [comment, marked as answer]
That's a bug. Created issue #130 to track.
